src/Shared/FileUtilitiesRegex.cs

```diff
@@ -34,5 +34,180 @@ internal static class FileUtilitiesRegex
             new Regex(
                 string.Format(CultureInfo.InvariantCulture, @"{0}$", s_baseUncPattern),
                 RegexOptions.Compiled);
```

Comment:
Are these regexes still used? Where?

Reply:
After this change they're entirely unused except for in tests.

Reply:
At this point I'm thinking maybe just pull the regexes into the test code so the product code is clean and no one's tempted to use them in the future?

Reply:
Done.